index.d.ts

```diff
@@ -0,0 +1,25 @@
+import { Plugin } from 'webpack';
+
+interface WebpackChunkHashPluginOptions {
+  /**
+   * A callback to add more content to the resulting hash
+   */
+  additionalHashContent?: (_: any) => string;
```

Comment:
Sure, I will update the PR with both your suggestions. Thanks for the fast feedback.